eng/testing/xunit/xunit.console.targets

```diff
@@ -87,11 +87,12 @@
     </ItemGroup>
 
     <!-- Workaround for https://github.com/xunit/xunit/issues/1651 -->
-    <ItemGroup Condition="'$(ArchiveTests)' != 'true'">
-      <None Remove="$(Pkgxunit_runner_visualstudio)\build\net452\xunit.runner.utility.net452.dll" />
-      <None Remove="$(Pkgxunit_runner_visualstudio)\build\net452\xunit.runner.reporters.net452.dll" />
-      <None Remove="$(Pkgxunit_runner_visualstudio)\build\netcoreapp2.1\xunit.runner.utility.netcoreapp10.dll" />
-      <None Remove="$(Pkgxunit_runner_visualstudio)\build\netcoreapp2.1\xunit.runner.reporters.netcoreapp10.dll" />
+    <ItemGroup Condition="'$(Pkgxunit_runner_visualstudio)' != '' AND '$(TargetFrameworkIdentifier)' == '.NETFramework'">
+      <!-- Remove `xunit.abstractions.dll` that was added to @(None) by xunit.runner.visualstudio -->
+      <_xunitAbstractionsRemove Include="@(None->WithMetadataValue('Filename', 'xunit.abstractions'))" />
+      <None Remove="@(_xunitAbstractionsRemove)" />
     </ItemGroup>
+    <Error Condition="'$(Pkgxunit_runner_visualstudio)' != '' AND '$(TargetFrameworkIdentifier)' == '.NETFramework' AND '@(_xunitAbstractionsRemove)' == ''"
```

Comment:
This condition will never be true. Even with an updated xunit.runner.visualstudio that contains the fix there's still a None item for xunit.abstractions.

What we fixed in xunit is the double signing of xunit.abstractions and by that the CopyToOutputDirectory="PreserveNewest" behavior.

Reply:
Seems a bit fragile to rely on the timestamp to avoid a double-write. Why can't Xunit just reference the packages for the dependencies it needs rather than redistributing them?